docs/blocks/administration/Team.story.tsx

```diff
@@ -75,7 +75,7 @@ export const TeamGeneral = () => {
 
   const checkboxTheme: CheckboxTheme = {
     ...defaultCheckboxTheme,
-    check: 'stroke-white',
+    checkMark: 'stroke-white',
```

Comment:
This is a break change - is it really nescarry ?

Reply:
not necessary,  will turn back